ansible.cfg

```diff
@@ -1,6 +1,12 @@
 [ssh_connection]
 pipelining=True
-#ssh_args = -F ./ssh-bastion.conf -o ControlMaster=auto -o ControlPersist=30m
+#ssh_args=-o ForwardAgent=yes
```

Comment:
The default option for ansible is "-o ControlMaster=auto -o ControlPersist=30m", so any example options should include them. Without it, the deployment is dramatically slower.

Reply:
So, I should take it out, or leave it? Confused :)

Reply:
I'm just suggesting that if you want to include any ssh_args (even commented), leave those two options in